src/globalComponents.spec.ts

```diff
@@ -1,9 +1,3 @@
 import { assert } from 'chai';
 import * as React from 'react';
-import "./globalComponents";
 
-describe('globalComponents', ()=>{
```

Comment:
thanks for pointing this out, I didn't notice you took this out. I disagree. Lets talk in person